admin/formatter/class-metabox-formatter.php

```diff
@@ -262,4 +262,17 @@ private function is_markdown_enabled() {
 		 */
 		return apply_filters( 'wpseo_is_markdown_enabled', $is_markdown );
 	}
+
+	/**
+	 * Checks whether word form recognition is active for the used language.
+	 *
+	 * @param string $language The used language.
+	 *
+	 * @return boolean Whether word form recognition is active for the used language.
+	 */
+	private function is_word_form_recognition_active( $language ) {
+		$supported_languages = [ 'de', 'en', 'es', 'fr', 'it', 'nl', 'ru' ];
+
+		return in_array( $language, $supported_languages, true );
+	}
```

Comment:
I think it's better to move this entire function to `WPSEO_Language_Utils`, as we might need this check in other places too.

Reply:
I wasn't sure where to put the function, but chose for this location with @andizer's anti-util crusade in mind. @andizer what do you think?

Reply:
I would keep it this way and convert it to a util when we needed it later on. Just to prevent one time use utils.